README.md

```diff
@@ -43,3 +40,19 @@ Please read our [Contributing Guide](CONTRIBUTING.md) for detailed information a
 
 - [Randstad Housing](resources/randstad-housing.md)
 - [Supportive Tech Organizations](resources/supportive-tech-orgs.md)
+
+## Support
+
+If you encounter any issues, have questions, or need assistance, please refer to our support [documentation](https://github.com/shesharpnl/.github/blob/main/SUPPORT.md).
+
+## Code of Conduct
+
+To ensure a welcoming and inclusive community, we have established a Code of Conduct that applies to everyone participating in our project. Please read and adhere to these [guidelines](https://github.com/shesharpnl/.github/blob/main/CODE_OF_CONDUCT.md).
+
+## Security
```

Comment:
I'm not sure if we should include security in the README of this repo, because we don't have potential security issue here. But it's your call! 😄

Reply:
I think we should leave this. :)